interp/interp_test.go

```diff
@@ -928,6 +928,10 @@ var runTests = []runTest{
 		`mkdir a; ln -s a b; [[ "$(cd a && pwd -P)" == "$(cd b && pwd -L)" ]]; echo $?`,
 		"1\n",
 	},
+	{
+		`orig="$PWD"; mkdir a && cd a && cd - && [[ "$PWD" == "$orig" ]]`,
```

Comment:
a bit easier/shorter without the &&, and it will still fail if the feature isn't working:

```suggestion
		`orig="$PWD"; mkdir a; cd a; cd -; [[ "$PWD" == "$orig" ]]`,
```